### ISSUE TITLE
Lots of Warning: Calling 'xxxxxx' is deprecated!

### ISSUE THREAD

**Issue by luisspuerto** [OSGeo/homebrew-osgeo4mac#306]
I don't know if you are aware of this: 

```
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:66:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:67:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/pyspatialite.rb:29:in `<class:Pyspatialite>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/orfeo5.rb:54:in `<class:Orfeo5>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :gpg' is deprecated!
Use 'depends_on "gnupg"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/qca-qt4.rb:43:in `<class:QcaQt4>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :postgresql' is deprecated!
Use 'depends_on "postgresql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:50:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mysql' is deprecated!
Use 'depends_on "mysql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:51:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/ossim.rb:32:in `<class:Ossim>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:66:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:67:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/pyspatialite.rb:29:in `<class:Pyspatialite>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/orfeo5.rb:54:in `<class:Orfeo5>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :gpg' is deprecated!
Use 'depends_on "gnupg"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/qca-qt4.rb:43:in `<class:QcaQt4>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :postgresql' is deprecated!
Use 'depends_on "postgresql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:50:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mysql' is deprecated!
Use 'depends_on "mysql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:51:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/ossim.rb:32:in `<class:Ossim>'
Please report this to the osgeo/osgeo4mac tap!
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:66:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:67:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/pyspatialite.rb:29:in `<class:Pyspatialite>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/orfeo5.rb:54:in `<class:Orfeo5>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :gpg' is deprecated!
Use 'depends_on "gnupg"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/qca-qt4.rb:43:in `<class:QcaQt4>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :postgresql' is deprecated!
Use 'depends_on "postgresql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:50:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mysql' is deprecated!
Use 'depends_on "mysql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:51:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/ossim.rb:32:in `<class:Ossim>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:66:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:67:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/pyspatialite.rb:29:in `<class:Pyspatialite>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/orfeo5.rb:54:in `<class:Orfeo5>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :gpg' is deprecated!
Use 'depends_on "gnupg"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/qca-qt4.rb:43:in `<class:QcaQt4>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :postgresql' is deprecated!
Use 'depends_on "postgresql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:50:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mysql' is deprecated!
Use 'depends_on "mysql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:51:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mpi' is deprecated!
Use 'depends_on "open-mpi"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/ossim.rb:32:in `<class:Ossim>'
Please report this to the osgeo/osgeo4mac tap!
```

**Comment by ansonhoyt** [comment]
Looks like this is because [Homebrew 1.5.0](https://brew.sh/2018/01/19/homebrew-1.5.0/) just deprecated `depends_on "python"` since the `python` formula will be upgraded to python 3 on March 31st. See the [release announcement](https://brew.sh/2018/01/19/homebrew-1.5.0/) for details.

**Comment by luisspuerto** [comment]
Ok!! I was guessing something like that, I just wanted to communicate the issue. 🙂

**Comment by luisspuerto** [comment]
By the way @ansonhoyt, does this mean that are we going to have the warnings till March 31st?

**Comment by 3nids** [comment]
See #308

**Comment by luisspuerto** [comment]
great @3nids ! I understand that a fix is on its way. 🙂

**Comment by nickrobison** [comment]
I think this has been resolved in #308.
